app/viewmodels/helpers/ViewMovementDocumentHelper.scala

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package viewmodels.helpers
+
+import models.response.emcsTfe.GetMovementResponse
+import play.api.i18n.Messages
+import play.twirl.api.{Html, HtmlFormat}
+import uk.gov.hmrc.govukfrontend.views.Aliases._
+import uk.gov.hmrc.govukfrontend.views.viewmodels.content.Text
+import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
+import utils.ExpectedDateOfArrival
+import viewmodels.govuk.TagFluency
+import views.html.components.{h2, summaryCard}
+
+import javax.inject.Inject
+
+class ViewMovementDocumentHelper @Inject()(h2: h2,
+                                           summaryCard: summaryCard,
+                                          ) extends ExpectedDateOfArrival with TagFluency {
+
+
+  def constructMovementDocument(movement: GetMovementResponse)(implicit messages: Messages): Html = {
+    val notProvidedMessage = messages("viewMovement.document.notProvided")
+    HtmlFormat.fill(Seq(h2(messages("viewMovement.document.details"))) ++
+      (movement.documentCertificate match {
+        case Some(value) => value.zipWithIndex.map {
+          case (document, index) =>
+            summaryCard(
+              card = Card(
+                Some(CardTitle(Text(messages("viewMovement.document.heading", index + 1)))),
+              ),
+              summaryListRows = Seq(
+                summaryListRowBuilder(
+                  "viewMovement.document.documentReference",
+                  document.documentReference.getOrElse(notProvidedMessage)
+                ),
+                summaryListRowBuilder(
+                  "viewMovement.document.documentType",
+                  document.documentType.getOrElse(notProvidedMessage)
```

Comment:
The document type comes back as an integer, I've asked UCD to see if this should be displayed as the integer or mapped to the 'friendly' name

Reply:
UCD confirmed that this should be the name of the document type not the document type itself